### PR TITLE
" = not needed to compile

### PR DESCRIPTION
**Pull request by Hughgent**
[why]: Lakilla asked for it.

still compiles and works.

### CODE REVIEW

**Comment by DominikPanic** [comment]
Should be GLOB lists but it's better than nothing.

File: code/__HELPERS/global_lists.dm [hunk 1]
```diff
@@ -12,11 +12,11 @@ var/global/list/active_diseases = list()
...
-var/global/list/metatips = file2list('config/tips/metatips.txt')
-var/global/list/marinetips = file2list('config/tips/marinetips.txt')
-var/global/list/xenotips = file2list('config/tips/xenotips.txt')
-var/global/list/joketips = file2list('config/tips/joketips.txt')
-var/global/list/alltips = marinetips + xenotips + joketips + metatips
+GLOBAL_LIST_INIT(metatips, world.file2list("config/tips/metatips.txt"))
+GLOBAL_LIST_INIT(marinetips, world.file2list("config/tips/marinetips.txt"))
+GLOBAL_LIST_INIT(xenotips, world.file2list("config/tips/xenotips.txt"))
+GLOBAL_LIST_INIT(joketips, world.file2list("config/tips/joketips.txt"))
+GLOBAL_LIST_INIT(alltips, GLOB.marinetips + GLOB.xenotips + GLOB.joketips + GLOB.metatips)
```
Comment (spookydonut) [inline code/__HELPERS/global_lists.dm]: why?

`#define ALLTIPS GLOB.marinetips + GLOB.xenotips + GLOB.joketips + GLOB.metatips`

File: code/__HELPERS/global_lists.dm [hunk 2]
```diff
@@ -12,11 +12,11 @@ var/global/list/active_diseases = list()
 var/global/list/events = list()
 
 //tips
-var/global/list/metatips = file2list('config/tips/metatips.txt')
-var/global/list/marinetips = file2list('config/tips/marinetips.txt')
-var/global/list/xenotips = file2list('config/tips/xenotips.txt')
-var/global/list/joketips = file2list('config/tips/joketips.txt')
-var/global/list/alltips = marinetips + xenotips + joketips + metatips
+GLOBAL_LIST_INIT(metatips, world.file2list("config/tips/metatips.txt"))
```
Comment (spookydonut) [inline code/__HELPERS/global_lists.dm]: please move these in to init rather than on world startup

File: code/__HELPERS/global_lists.dm (hunk 1 above)
Comment (Hughgent) [reply code/__HELPERS/global_lists.dm]: I presume you want that line to be that.

File: code/controllers/configuration/configuration.dm [hunk 3]
```diff
@@ -20,6 +20,11 @@
 
 	var/motd
 
+/datum/controller/configuration/proc/loadtips()
+	GLOB.metatips = world.file2list("config/tips/metatips.txt")
```
Comment (spookydonut) [inline code/controllers/configuration/configuration.dm]:
```suggestion
	GLOB.metatips = world.file2list("[directory]/tips/metatips.txt")
```

Comment (spookydonut) [reply code/controllers/configuration/configuration.dm]: also add some `log_config()` if the files dont exist or are empty

**Pull request merged**